ExilenceNextApp/src/components/toolbar/Toolbar.tsx

```diff
@@ -266,6 +268,23 @@ const Toolbar = ({
                   </IconButton>
                 </span>
               </Tooltip>
+              <Tooltip title={t('label.remove_snapshot_icon_title') || ''} placement="bottom">
+                <span>
+                  <IconButton
+                    disabled={
+                      !activeProfile ||
+                      isSnapshotting ||
+                      !signalrOnline ||
+                      activeProfile.snapshots.length === 0
+                    }
+                    onClick={() => handleClearSnapshot()}
```

Comment:
`onClick={() => handleClearSnapshot()}` can be shortened to `onClick={handleClearSnapshot}`